aten/src/ATen/native/quantized/cpu/qadd.cpp

```diff
@@ -249,6 +249,16 @@ Tensor qadd_scalar_out(Tensor qa, Scalar b, Tensor out) {
   return _add_scalar_out<ReLUFused>(out, qa, b);
 }
 
+template <bool ReLUFused = false>
+Tensor qadd_scalar_tensor(Tensor qa, Tensor b) {
```

Comment:
wondering if there is context for this...b is a tensor which represents a scalar?  Is this related to how tracing works?  Maybe we can add a comment?

Reply:
yeah, will add a comment. tracing will trace a Scalar as tensor

Reply:
makes sense, thx!

Reply:
Wondering if the comment is accurate: add and add_scalar cannot be merged as add_scalar does not require a scale and zero-point.

Reply:
it's a little bit confusing, I meant when we add boradcasting op, we can merge add_scalar with that op, a separate task is to merge all variations of quantized::add into `quantized::add`, since we have overloading.